lang/LangSource/PyrLexer.cpp

```diff
@@ -1762,21 +1762,12 @@ void traverseFullDepTree2()
 		} else {
 			double elapsed;
 			buildBigMethodMatrix();
-			SymbolTable* symbolTable = gMainVMGlobals->symbolTable;
-			//post("\tNumber of Symbols %d\n", symbolTable->NumItems());
-			//post("\tByte Code Size %d\n", totalByteCodes);
-			//elapsed = TickCount() - compileStartTime;
-			//elapsed = 0;
-			elapsed = elapsedTime() - compileStartTime;
-			//post("\tcompiled %d files in %.2f seconds\n",
-			//	 gNumCompiledFiles, elapsed );
 			if(numOverwrites == 1){
 				post("\nInfo: One method is currently overwritten by an extension. To see which, execute:\nMethodOverride.printAll\n\n");
 			}
 			else if(numOverwrites > 1){
 				post("\nInfo: %i methods are currently overwritten by extensions. To see which, execute:\nMethodOverride.printAll\n\n", numOverwrites);
 			}
-			//post("compile done\n");
```

Comment:
see again those tests I posted from supercollider.js

I'm not sure this is the only place it decides that it's really done.

it is absolutely essential that this is posted. not just for supercollider.js but for all human beings who need to know if it the darned thing compiled. that tells you that the next bunch of crap it dumps is something happening during class tree init.